backend/main_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func mustTime(s string) time.Time {
+	t, err := time.Parse("2006-01-02", s)
+	if err != nil {
+		panic(err)
+	}
+	return t
+}
+
+func TestParse(t *testing.T) {
+	s := `
+Date, 	    Coutry, Province, Lat, Long, Confirmed, Recovered, Deaths
```

Comment:
this has a tab in it, which makes it not line up on my screen, which uses 8 spaces for tabs as everything should.

Also, Coutry is a mispelling of Country

Reply:
sorry, fixed.